src/main/java/org/osiam/addons/self_administration/registration/UserConverter.java

```diff
@@ -100,9 +100,14 @@ public User toScim(RegistrationUser registrationUser) {
                 .addPhoneNumbers(getPhoneNumberList(registrationUser))
                 .addIms(getImList(registrationUser))
                 .addPhotos(getPhotoList(registrationUser))
-                .addAddresses(getAddressList(registrationUser))
                 .addExtensions(getExtensions(registrationUser));
 
+        final List<Address> addresses = getAddresses(registrationUser);
+
+        if (addresses.size() > 0) {
+            userBuilder.addAddresses(addresses);
+        }
```

Comment:
removing the last 5 lines and restoring [line 103](https://github.com/osiam/addon-self-administration/pull/114/files#diff-48288e5fa146590d9e691bcbf4132322L103) still yields green tests. is it really necessary to make the process that handles addresses different than the processes for all the other multi-valued attributes?